docs/site/_includes/getting_started/global/partials/NOTICES_ENVIRONMENT.liquid

```diff
@@ -1,9 +1,9 @@
 {%- if page.platform_type == "cloud" %}
 {% alert level="warning" %}
 {%- if page.lang == "ru" %}
-Перед началом установки убедитесь в наличии квот облачного провайдера, необходимых для развертывания вашего кластера.{%- if page.platform_code == "aws" %} Проверьте наличие доступа к необходимым регионам.{%- endif %}{%- if page.platform_code == "gcp" %} Убедитесь, что у вас включен [Compute Engine API](https://console.cloud.google.com/marketplace/product/google/compute.googleapis.com).{%- endif %}
+Перед началом установки убедитесь в наличии {% if page.platform_code == "zvirt" %}ресурсов гипервизора{%- else %}квот облачного провайдера{% endif %}, необходимых для развертывания вашего кластера.{%- if page.platform_code == "aws" %} Проверьте наличие доступа к необходимым регионам.{%- endif %}{%- if page.platform_code == "gcp" %} Убедитесь, что у вас включен [Compute Engine API](https://console.cloud.google.com/marketplace/product/google/compute.googleapis.com).{%- endif %}
```

Comment:
I think there is no need to separate zvirt and the other cloud providers because zvirt can be installed on multiple hypervisors